plinth/modules/dynamicdns/templates/dynamicdns_status.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+{% comment %}
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+{% endcomment %}
+
+{% load bootstrap %}
+
+{% block content %}
+  <p>
+  <h3>NAT type</h3>
+  {% if nat_unchecked %}
+  	NAT type not detected yet, if you do not provide a "IP check URL" we will 
```

Comment:
There are spaces and tabs mixed for indentation here. Depending on the editor this should be pretty easy to change, for vim it's just `:retab`.

So far we use 2 spaces to indent html and javascript (as you are doing, except for the tabs)